pkg/networkservice/common/heal/client.go

```diff
@@ -32,20 +33,22 @@ import (
 
 type healClient struct {
 	chainCtx        context.Context
-	livelinessCheck LivelinessCheck
+	livenessChecker LivenessChecker
+	attemptAfter    time.Duration
```

Comment:
Why do we need this?

Reply:
To make sandbox tests run faster. In the latest revision of this PR replaced with the `BackOff` interface.